### PR TITLE
Fix bugs in webportal deployment

### PR DESCRIPTION
**Pull request by abuccts**
Fix bug in webportal deployment:
- Always pull docker image.
- Use `yarn install` instead of `npm install`.

### CODE REVIEW

File: service-deployment/bootstrap/webportal/webportal-ds.yaml.template [hunk 1]
```diff
@@ -33,6 +33,9 @@ spec:
       containers:
       - name: webportal
         image: {{ clusterinfo['dockerregistryinfo']['prefix'] }}webportal
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
```
Comment (hwuu) [inline service-deployment/bootstrap/webportal/webportal-ds.yaml.template]: Does this mean the docker command will use the `--privileged` parameter? I heard from @fanyangCS that this is not secure.

Comment (hwuu) [inline service-deployment/bootstrap/webportal/webportal-ds.yaml.template]: Does this mean the docker command will use the `--privileged` parameter? I heard from @fanyangCS that this is not secure.

File: service-deployment/src/webportal/dockerfile [hunk 2]
```diff
@@ -24,7 +24,7 @@ ENV NODE_ENV=production \
 
 COPY copied_file/ /usr/src/
 COPY copied_file/webportal/ /usr/src/app/
-RUN npm install
+RUN npm run yarn install
```
Comment (hwuu) [inline service-deployment/src/webportal/dockerfile]: Is it really necessary to use yarn? I tried `npm install` and it worked fine.

Comment (abuccts) [reply service-deployment/src/webportal/dockerfile]: of course, we use yarn as package manager.
you can find the doc [here](https://yarnpkg.com/en/docs/migrating-from-npm)

Comment (fanyangCS) [reply service-deployment/src/webportal/dockerfile]: [https://www.sitepoint.com/yarn-vs-npm/](https://www.sitepoint.com/yarn-vs-npm/) here are more info about why yarn is better than npm

---
In reply to: [158422658](https://github.com/Microsoft/pai/pull/124#discussion_r158422658) [](ancestors = 158422658)

**Pull request merged**